### PR TITLE
Answer:8

### PR DESCRIPTION
**Pull request by tom-englert**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: apps/pipe-easy/src/app/app.component.ts [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
   `,
 })
 export class AppComponent {
   persons = ['toto', 'jack'];
 
-  heavyComputation(name: string, index: number) {
-    // very heavy computation
-    return `${name} - ${index}`;
+  constructor(cdr: ChangeDetectorRef) {
+    setInterval(() => cdr.markForCheck(), 1000);
```
Comment (tomalaforge) [inline apps/pipe-easy/src/app/app.component.ts:34]: I think this is only for testing purposes ? 🧐

File: apps/pipe-easy/src/app/app.component.ts [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
+  PipeTransform,
+} from '@angular/core';
+
+@Pipe({
+  standalone: true,
+  pure: true,
+  name: 'heavyComputation',
+})
+export class HeavyComputationPipe implements PipeTransform {
+  transform(value: string, ...args: [index: number]) {
```
Comment (tomalaforge) [inline apps/pipe-easy/src/app/app.component.ts:15]: Why did you write `...args: [index: number]` this way ? 

and not only `index: number`

File: apps/pipe-easy/src/app/app.component.ts (hunk 1 above)
Comment (tom-englert) [reply apps/pipe-easy/src/app/app.component.ts:34]: Yes, just to see that the pipe output does not change

File: apps/pipe-easy/src/app/app.component.ts (hunk 2 above)
Comment (tom-englert) [reply apps/pipe-easy/src/app/app.component.ts:15]: You are right, that can be simplified.
I was already thinking about how to solve the typed challenge 😃

**Pull request closed without merging**